litestar/middleware/session/client_side.py

```diff
@@ -38,12 +38,19 @@
 NONCE_SIZE = 12
 CHUNK_SIZE = 4096 - 64
 AAD = b"additional_authenticated_data="
+SET_COOKIE_INCLUDE = {f.name for f in fields(Cookie) if f.name not in {"key", "secret"}}
+CLEAR_COOKIE_INCLUDE = {f.name for f in fields(Cookie) if f.name not in {"key", "secret", "max_age"}}
```

Comment:
any benefits / obvious performance increase if try to use `SET_COOKIE_INCLUDE` here since its already filtered on "key" and "secret"?

Reply:
You could do `CLEAR_COOKIE_INCLUDE = SET_COOKIE_INCLUDE - {"max_age"}`.

> any benefits

Is it clearer? IDK, maybe.

> obvious performance increase

No, its only done once on import.

Feel free to PR this change into the branch if you like.